app/views/games/show.html.erb

```diff
@@ -1,3 +1,29 @@
+<script>
+  $(function() {
+    $('.piece').draggable();
+    $('td').droppable({
+      drop: function(ev, ui) {
+        var valid_move = false;
+
+        $.ajax({
+          type: 'PUT',
+          async: false,
```

Comment:
hmm I would expect `fail()` to work.

If your rails controller returns a `success` status. such as `render status: :ok`. Then it should run the `done` code.

If it fails, have it return `render status: :bad_request`. And it should run the `fail` code.

I haven't used jquery in a while. Here is some example code I stole.
```JavaScript
var subscribeRequest = $.ajax({
     type: "POST",
     url: "subscribe.php",
     data: { subscriberName: $('#name').val(), emailAddress: $('#email').val() }
});

subscribeRequest.done(function(msg)
{
     alert( "You have successfully subscribed to our mailing list." );
});

subscribeRequest.fail(function(jqXHR, textStatus)
{
     alert( "We could not subscribe you please try again or contact us if the problem persists (" + textStatus + ")." );
});
```
from: http://www.sitepoint.com/5-jquery-ajax-examples/